src/manifest.json

```diff
@@ -7,7 +7,7 @@
   "content_scripts": [{
     "css": ["style.css"],
     "js": ["jquery.min.js", "content.js"],
-    "matches": ["https://github.com/*"]
+    "matches": ["https://github.com/peckadesign/*/pull/*"]
```

Comment:
To nevím jestli je uplně ideální, protože tam bývala i funkce, která na detailu issue vytahovala ty jednotlivé komentáře s odevzdaným úkolem a dávala to do toho pluginu viz. https://github.com/peckadesign/PeckaGitHubPlugin/blob/master/src/content.js#L73

Ale bohužel je tam asi nějaký šotek a nefunguje to teďka. Tak nevím co s tím, buď můžeš v rychlosti kouknout proč to nefunguje na tom issue a fixnout to a přidat i ty issue tady do toho pole. Nebo je tam asi zatím nedat dokud to někdo nefixne.